client/python/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "armada_client"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
We need to decide on what to put here. Maybe "G Research"? What do others in the community do?

Reply:
I put my email for now.  Waiting to hear from people.